Logger.Tests/FileLoggerTests.cs

```diff
@@ -1,10 +1,47 @@
 ﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using System.IO;
 
 namespace Logger.Tests
 {
     [TestClass]
     public class FileLoggerTests
     {
-        
+        [TestMethod]
+        public void CreateFileLogger_Success()
+        {
+            _ = new FileLogger("");
+        }
+
+        [TestMethod]
+        [ExpectedException(typeof(ArgumentNullException))]
+        public void LogPath_SetToNullInConstructor_ThrowArgumentNullException()
+        {
+            _ = new FileLogger(null!);
```

Comment:
```suggestion
            _ = new FileLogger(null);
```
See my comment on line 25 in `FileLogger.cs` for why you can now remove this.